interpreter/cling/tools/plugins/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+#------------------------------------------------------------------------------
+# CLING - the C++ LLVM-based InterpreterG :)
```

Comment:
There is a typo above, and I belive the smiley is a bit out of place.

Reply:
Sorry, I cannot see the typo. Could you be more specific. About the smiley it is courtesy of @Axel-Naumann.

Reply:
`InterpreterG`